README.md

```diff
@@ -27,16 +27,16 @@ What is the purpose of the `alt` attribute? e.g:
 <img src='https://jesse.sh/img/me.jpg' alt='Jesse Shawl'/>
 ```
 
-```text
-Your answer here.
+```
+To facilitate understanding for those who are blind.  So if I am blind the website it able to know the picture that is being displayed on the screen/ web page.
 ```
```

Comment:
Yes, but also if the user is unable to view the image for whatever reason (maybe the img link is broken), the alt attribute will display the text of the image to indicate it's content.